rudder-inventory-ldap/debian/postinst

```diff
@@ -65,6 +65,10 @@ case "$1" in
 					exit 1
 				fi
 
+        # Reload systemd when it is available so that it creates sysv-init shims
+        # This is necessary to make the service command work, and will be obsolete when we create systemd units
+        [ -x /bin/systemctl ] && /bin/systemctl daemon-reload
```

Comment:
this should also be of the if block - error at install is between INFO: Restarting syslogd... and INFO: Restarting rudder-slapd... , so not in this block. It's probably safer to add it on line 40